src/scalper.ts

```diff
@@ -610,7 +610,10 @@ export class Scalper {
 
     // Settle Funds
     try {
-      await settleOpenBook(this.connection, this.owner, spotMarket, this.symbol, 'USDC');
+      const settleTransaction = await settleOpenBook(this.connection, this.owner, spotMarket, this.symbol, 'USDC');
+      if (settleTransaction != undefined) {
```

Comment:
add a comment explaining what an undefined response means. does undefined mean that it was a success or not?

Reply:
ideally, we dont rely on undefined and have an explicit success or failure value

Reply:
Only returns a tx instruction if there are funds to settle, added comment